getting_started.md

```diff
@@ -0,0 +1,151 @@
+
+We keep a bootstrap script that handles installation in our supported packages publicly available, you can get it and run it by running the following command:
+
+```bash
+$ curl -L http://packages.overwatchsec.com/bootstrap.sh | bash
+```
+
+In order to keep communications between devices and Overwatch secure, each device is identified through an ssh key created at install time and which will be shown in your device after running the bootstrap script; you can then either register the device in the overwatch web wizard by copy/pasting the ssh key into the UI, or you can use the agent itself for device registration with it's [command line interface](#agent-command-line-usage)
+
+Once the registration is complete your device will be able to connect to our servers, start sending health reports and download and enforce any security policies you set up for it.
+
+### Agent command line usage
+
+The Overwatch agent has a command line interface that allows you to perform basic administrative tasks, you should have the executable available as `overwatchd` in your `PATH`.
+
+#### Register agent through CLI.
+
+The agent can register itself without the need to access the Overwatch web.
+
+##### Register interactively
+
+```bash
+$ sudo overwatchd --register --name "Media Center Raspberry Pi"
+```
+
+This will prompt an interactive email/password input and register the device under the ownership of the authenticating user, the `--name` flag is optional, we'll register the device under a made-up name if it's not specified.
+
+##### Register non-interactively
+
+Alternatively, you can specify a `--token <token>` to skip the interactive prompt, this token is associated to your account and will also register the device under your user, this is specially useful when installing the agent in bulk in multiple devices.
+
+```bash
+$ sudo overwatchd --register --name "Smoke detector" --token Xdj2kdlajk3dfjsk2j31hdlrgk3od12isdjiqk2m22i3jdfui
+```
+
+##### Get registration token
+
+You can get your registration token at any time with the `--get-option` flag, it will ask for login credentials interactively.
+
+```bash
+$ overwatchd --get-token
+```
+
+##### Regenerating identity file.
+
+At any point you might want to recreate the ssh identity keys, if you do you will need to register the device again, as it will appear to Overwatch as a completely new and unrelated device.
+
+```bash
+$ sudo overwatchd --generate-keys --override
+```
+
+#### Starting the agent manually.
+
+The Overwatch agent is installed as a service in your device, as such, you'll be able to issue standard service commands to control it's behavior.
+
+```bash
+$ sudo service start overwatchd
+$ sudo service restart overwatchd
+$ sudo service stop overwatchd
+```
+
+#### Supported Devices
+
+Overwatch targets IoT platforms so this list will hopefully expand in the near future, our bootstrap script currently supports:
+
+* upstart-based Ubuntu Linux (by default, Ubuntu 14.10 and older).
```

Comment:
We currently support upstart, there's a PR waiting to be merged with systemd support (I'll update this document when merged)